spec.md

```diff
@@ -146,33 +145,23 @@ Messages can be delivered through various industry standard protocol (e.g. HTTP,
 AMQP, MQTT, SMTP), open-source protocols (e.g. Kafka, NATS), or platform/vendor
 specific protocols (AWS Kinesis, Azure Event Grid).
 
-## Type System
+## Context Attributes
+
+Every CloudEvent conforming to this specification MUST include context
+attributes designated as REQUIRED and MAY include one or more OPTIONAL context
+attributes.
+
+These attributes, while descriptive of the event, are designed such that they
+can be serialized independent of the event data. This allows for them to be
+inspected at the destination without having to deserialize the event data.
 
 The following abstract data types are available for use in attributes. Each of
 these types MAY be represented differently by different event formats and in
 transport metadata fields. This specification defines a canonical
 string-encoding for each type that MUST be supported by all implementations.
 
-A strongly-typed programming model that represents a CloudEvent or any extension
-MUST be able to convert from and to the canonical string-encoding to the
-runtime/language native type that best corresponds to the abstract type.
-
-For example, the `time` attribute might be represented by the language's native
-_datetime_ type in a given implementation, but it MUST be settable providing an
-RFC3339 string, and it MUST be convertible to an RFC3339 string when mapped to a
-header of an HTTP message.
-
-A CloudEvents transport binding or event format implementation MUST likewise be
-able to convert from and to the canonical string-encoding to the corresponding
-data type in the encoding or in transport metadata fields.
-
-An attribute value of type `Timestamp` might indeed be routed as a string
-through multiple hops and only materialize as a native runtime/language type at
-the producer and ultimate consumer. The `Timestamp` might also be routed as a
-native transport type and might be mapped to/from the respective
-language/runtime types at the producer and consumer ends, and never materialize
-as a string.
-
+- `Boolean` - a boolean value of "true" or "false".
```

Comment:
Did you mean to add `Boolean` to the type system in this PR?

Reply:
yes I added it because someone may want to define an extension of type boolean. I pointed it out on the call and there was no objection.